requirements-dev.txt

```diff
@@ -4,6 +4,7 @@ pytest==3.0.7
 pytest-cov==2.4.0
 python-coveralls==2.5.0
 tox==2.1.1
+blinker==1.4
```

Comment:
I don't want to add this to setup.py and want blinker to be optional. This should be an optional feature and the user would need to install blinker separately. Thus, we have that try/catch around the import.